frontend/public/components/utils/copy-to-clipboard.tsx

```diff
@@ -19,7 +19,12 @@ export const CopyToClipboard: React.FC<CopyToClipboardProps> = React.memo((props
 
   return (
     <div className="co-copy-to-clipboard">
-      <pre className="co-pre-wrap co-copy-to-clipboard__text">{visibleValue}</pre>
+      <pre
+        className="co-pre-wrap co-copy-to-clipboard__text"
+        data-test="co-copy-to-clipboard__text"
```

Comment:
```suggestion
        data-test="copy-to-clipboard"
```
Data test doesn't need to adhere BEM naming convention.